src/connectors/greathorn_logs.py

```diff
@@ -7,12 +7,14 @@
 import os
 import requests
 import time
+from typing import Any
 
 import fire
 from dateutil.parser import parse as parse_date
 
-from connectors.utils import Bearer
+from connectors.utils import Bearer, yaml_dump
 from runners.helpers import db, log
+from runners.helpers.dbconfig import ROLE
```

Comment:
extra import?

Reply:
It produces `connectors/greathorn_logs.py:65: error: Name "ROLE" is not defined` otherwise actually, because of [line 64](https://github.com/snowflakedb/SnowAlert/blob/26e265c960806ded9ec1bbc482933fd2cb5c129c/src/connectors/greathorn_logs.py#L64)

Reply:
yikes, good catch